### PR TITLE
Season stat edge test

### PR DESCRIPTION
**Pull request by thawley2**
Created edge case testing for the SeasonStats class.  Added methods to check if the season is available and return an error message if it is not.

### CODE REVIEW

**Review by CrowRising — APPROVED** [review]
Nice work, good idea for edge cases!

File: lib/season_statables.rb [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+    seasons = @game_teams.map(&:season_id)
+    if !seasons.include?(season)
+     true
+    end
+  end
+
+  def invalid_season
+    'Invalid Season selection'
+  end
+
```
Comment (andrew-bingham1) [inline lib/season_statables.rb:13]: Really clever idea for testing those edge cases!

File: spec/season_stats_spec.rb [hunk 2]
```diff
@@ -97,15 +101,27 @@
...
   end
+
+  describe 'error message' do
+    it '#season_verification' do
+      expect(@season_stat.season_not_found?(20202021)).to eq true
+    end
+
+    it '#invalid_season' do
+      expect(@season_stat.invalid_season).to eq('Invalid Season selection')
+    end
```
Comment (andrew-bingham1) [inline spec/season_stats_spec.rb:125]: Same here, nice job figuring this out

**Review by andrew-bingham1 — APPROVED** [review]
Thanks for coming up with ways to test those edge cases!!

**Review by angelbyun — APPROVED** [review]
The edge testing looks great. Thanks for going in there and getting that done!
